cpp/models/ode_secirts/model.h

```diff
@@ -0,0 +1,971 @@
+            size_t INSIIi  = this->populations.get_flat_index({i, InfectionState::InfectedNoSymptomsImprovedImmunity});
+            size_t ISyIIi  = this->populations.get_flat_index({i, InfectionState::InfectedSymptomsImprovedImmunity});
+            size_t ISevIIi = this->populations.get_flat_index({i, InfectionState::InfectedSevereImprovedImmunity});
+            size_t ICrIIi  = this->populations.get_flat_index({i, InfectionState::InfectedCriticalImprovedImmunity});
+
+            size_t INSIICi =
+                this->populations.get_flat_index({i, InfectionState::InfectedNoSymptomsImprovedImmunityConfirmed});
+            size_t ISyIICi =
+                this->populations.get_flat_index({i, InfectionState::InfectedSymptomsImprovedImmunityConfirmed});
+            size_t TImm1 = this->populations.get_flat_index({i, InfectionState::TemporaryImmunPartialImmunity});
+            size_t TImm2 = this->populations.get_flat_index({i, InfectionState::TemporaryImmunImprovedImmunity});
+
+            size_t SIIi = this->populations.get_flat_index({i, InfectionState::SusceptibleImprovedImmunity});
+
+            FP reducExposedPartialImmunity  = params.template get<ReducExposedPartialImmunity<FP>>()[i];
+            FP reducExposedImprovedImmunity = params.template get<ReducExposedImprovedImmunity<FP>>()[i];
+            FP reducInfectedSymptomsPartialImmunity =
+                params.template get<ReducInfectedSymptomsPartialImmunity<FP>>()[i];
+            FP reducInfectedSymptomsImprovedImmunity =
+                params.template get<ReducInfectedSymptomsImprovedImmunity<FP>>()[i];
+            FP reducInfectedSevereCriticalDeadPartialImmunity =
+                params.template get<ReducInfectedSevereCriticalDeadPartialImmunity<FP>>()[i];
+            FP reducInfectedSevereCriticalDeadImprovedImmunity =
+                params.template get<ReducInfectedSevereCriticalDeadImprovedImmunity<FP>>()[i];
+            FP reducTimeInfectedMild = params.template get<ReducTimeInfectedMild<FP>>()[i];
+
+            //symptomatic are less well quarantined when testing and tracing is overwhelmed so they infect more people
+            auto riskFromInfectedSymptomatic =
+                smoother_cosine(test_and_trace_required, params.template get<TestAndTraceCapacity<FP>>(),
+                                params.template get<TestAndTraceCapacity<FP>>() * 15,
+                                params.template get<RiskOfInfectionFromSymptomatic<FP>>()[i],
+                                params.template get<MaxRiskOfInfectionFromSymptomatic<FP>>()[i]);
+
+            auto riskFromInfectedNoSymptoms =
+                smoother_cosine(test_and_trace_required, params.template get<TestAndTraceCapacity<FP>>(),
+                                params.template get<TestAndTraceCapacity<FP>>() * 2,
+                                params.template get<RelativeTransmissionNoSymptoms<FP>>()[i], 1.0);
```

Comment:
factors 2 and 15 were made variable in the mean time in the vvs model. Could you adapt? And maybe also check if there were some other adaptations, in particular https://github.com/SciCompMod/memilio/issues/982 and https://github.com/SciCompMod/memilio/pull/1104